docs/usage/utils/subscribeToTransactionsWithProofs.md

```diff
@@ -1,5 +1,6 @@
 **Usage**: `async client.subscribeToTransactionsWithProofs(bloomFilter, options = { count: 0 })`
-**Description**:
+**Description**: From a provided bloomfilter, will returns a ClientReadableStream streaming the transaction matching the filter.
```

Comment:
```suggestion
**Description**: For any provided bloomfilter, it will return a ClientReadableStream streaming the transaction matching the filter.
```